quartz/components/RecentNotes.tsx

```diff
@@ -56,18 +58,20 @@ export default ((userOpts?: Partial<Options>) => {
                       <Date date={getDate(cfg, page)!} locale={cfg.locale} />
                     </p>
                   )}
-                  <ul class="tags">
-                    {tags.map((tag) => (
-                      <li>
-                        <a
-                          class="internal tag-link"
-                          href={resolveRelative(fileData.slug!, `tags/${tag}` as FullSlug)}
-                        >
-                          {tag}
-                        </a>
-                      </li>
-                    ))}
-                  </ul>
+                  {opts.showTags && (
+                    <ul class="tags">
+                      {tags.map((tag) => (
```

Comment:
how does the spacing look with the tags removed?

Reply:
i just added some screen caps to the pull request